### DISCUSSION TITLE
Paperles LXC Ubuntu Repos?

### DISCUSSION THREAD

**Discussion by DuschdrBabbe** [tteck/Proxmox#2157] [Help Needed]
Hello,
i got errors when i do apt update on this LXC:

`Ign:1 http://archive.ubuntu.com/ubuntu kinetic InRelease
Ign:2 http://archive.ubuntu.com/ubuntu kinetic-updates InRelease         
Ign:3 http://archive.ubuntu.com/ubuntu kinetic-security InRelease        
Err:4 http://archive.ubuntu.com/ubuntu kinetic Release                   
  404  Not Found [IP: 91.189.91.81 80]
Err:5 http://archive.ubuntu.com/ubuntu kinetic-updates Release
  404  Not Found [IP: 91.189.91.81 80]
Err:6 http://archive.ubuntu.com/ubuntu kinetic-security Release
  404  Not Found [IP: 91.189.91.81 80]
Reading package lists... Done
E: The repository 'http://archive.ubuntu.com/ubuntu kinetic Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'http://archive.ubuntu.com/ubuntu kinetic-updates Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'http://archive.ubuntu.com/ubuntu kinetic-security Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
root@paperless-ngx:~# `

**Comment by t0rb3n** [comment]
Are you running Ubuntu 22.10? 
Take a look at [this](https://askubuntu.com/questions/1458489/ubuntu-22-10-kinetic-cant-update-archive-ubuntu-com-and-ignored-it)
